index.html

```diff
@@ -4,16 +4,63 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width" />
     <title>Awesome Books</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <link rel="stylesheet" href="./styles.css">
   </head>
-
   <body>
-    <h1>Awesome Books</h1>
-    <div id="books"></div>
-    <form id="form">
-      <input id="title" type="text" placeholder="Title" />
-      <input id="author" type="text" placeholder="Author" />
-      <button id="add-button" type="submit">Add</button>
-    </form>
+    <header class ="header">
+      <nav class="container-l navbar navbar-expand-lg navbar-light bg-light p-3">
+        <h1 class="col-lg-9 h3">Awesome Books</h1>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarText" aria-controls="navbarText" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse mr-auto col-lg-3 justify-content-around" id="navbarText">
+          <ul class="navbar-nav">
+            <li class="nav-item">
+              <a id="list-nav" onclick="show('list-section')" class="nav-link">List</a>
+            </li>
+            <li class="nav-item">
+              <a id="add-nav" onclick="show('form-section')" class=" nav-link">Add New</a>
+            </li>
+            <li class="nav-item">
+              <a id="contact-nav" onclick="show('contact-section')" class=" nav-link">Contact</a>
+            </li>
+          </ul>
```

Comment:
- [ ] OPTIONAL
- The right menu elements in your navbar is missing the dividers between each menu element, as can be seen in the provided [wireframe design](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md). Kindly ensure to improve upon this aspect in your design, as we really want our designs to as much as possible mimic the provided wireframe template.
- To enhance the user experience of this design kindly ensure that an active menu link is highlighted(color change or change in opacity). In other words, the current active page should have the corresponding menu item highlighted.

**Your design menu:**
![image](https://user-images.githubusercontent.com/12745474/129422930-50072b78-9089-42ff-a809-f07c51071894.png)

**The wireframe menu design:**
![image](https://user-images.githubusercontent.com/12745474/129183812-a30acaa9-4f6c-4d4e-973b-5dd83a591c8d.png)